.storybook/stories-from-readme.js

```diff
@@ -66,7 +66,9 @@ export function addPlaygroundStory(playgroundModule) {
 function AppProviderDecorator(story) {
   return (
     <div style={{padding: '8px'}}>
-      <AppProvider i18n={en}>{story()}</AppProvider>
+      <React.StrictMode>
```

Comment:
Instead of setting this in the AppProviderDecorator I think we should add a new [global decorator](https://storybook.js.org/docs/addons/introduction/#storybook-decorators) to make sure this applies to all stories. (Admittedly currently AppProvider ends up being applied to everything but at some point that might change if we want to pull in stories from places other than the readmes)

Can you revert this and then in .storybook/config.js under the `addParameters` call add:

```
const StrictModeDecorator = story => <React.StrictMode>{story()}</React.StrictMode>;
addDecorator(StrictModeDecorator);
```